src/Simplex/Chat/Protocol.hs

```diff
@@ -161,7 +184,11 @@ instance ToJSON MsgRef where
   toJSON = J.genericToJSON J.defaultOptions {J.omitNothingFields = True}
   toEncoding = J.genericToEncoding J.defaultOptions {J.omitNothingFields = True}
 
-data ChatMessage e = ChatMessage {msgId :: Maybe SharedMsgId, chatMsgEvent :: ChatMsgEvent e}
+data ChatMessage e = ChatMessage
+  { chatVRange :: Maybe VersionRange,
```

Comment:
this doesn't need to be maybe, the absence of version in JSON should be interpreted as specific version, that would simplify the logic in other places.

Reply:
in the db it's also not nullable